app/Http/Controllers/Api/ActivityController.php

```diff
@@ -27,9 +27,17 @@ public function index(Request $request)
     {
         // Create an empty Signup query and eager-load posts, which we
         // can either filter or paginate to retrieve all signup records.
-        $query = $this->newQuery(Signup::class)->with('posts');
+        if ($request->query('orderByPost') === 'desc') {
+            // $query = $this->newQuery(Signup::class)->with('posts')->orderBy('posts.created_at', 'desc');
```

Comment:
will get rid of this, just wanted to keep for now in case we want to add this back in and go with this order instead!

Reply:
I'm trying to figure out the difference between these two queries. Does the first one just order the posts under the signups in reverse chronological order, and the second (uncommented-out one) actually order the _signups_ in order of which ones own the most recent posts?

Reply:
oh sorry @katiecrane ! So the first one was just supposed to be  $query = $this->newQuery(Signup::class)->with('posts')->orderBy('created_at', 'desc');` (I was trying to see if `posts.created_at` worked but it didn't!). In that case, that just orders all signups with posts in descending order.

And yes, you're exactly right for the uncommented-out query!

Reply:
Okay I think the uncommented-out one makes sense to me!